database/seeds/UsersTableSeeder.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class UsersTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+      \App\User::create(array(
+        'first_name' => 'Swornim',
+        'last_name'=> 'Shrestha',
+        'email' => 'srestaswrnm@gmail.com',
```

Comment:
use [Faker Library](https://github.com/fzaninotto/Faker) to generate random data. There are faker formatter for Nepali names as well.